src/pages/theming/css-shadow-parts.md

```diff
@@ -0,0 +1,153 @@
+---
+previousText: 'CSS Variables'
+previousUrl: '/docs/theming/css-variables'
+nextText: 'Colors'
+nextUrl: '/docs/theming/colors'
+---
+
+# CSS Shadow Parts
+
+CSS Shadow Parts allow developers to style CSS properties on an element inside of a shadow tree. This is extremely useful in customizing Ionic Framework <a href="https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM" target="_blank" rel="noopener noreferrer">Shadow DOM</a> components.
+
+## Why Shadow Parts?
+
+Ionic Framework is a distributed set of <a href="https://www.webcomponents.org/introduction" target="_blank" rel="noopener noreferrer">Web Components</a>. Web Components follow the <a href="https://w3c.github.io/webcomponents/spec/shadow/" target="_blank" rel="noopener noreferrer">Shadow DOM specification</a> in order to encapsulate styles and markup.
```

Comment:
Probably link to mdn web component docs: https://developer.mozilla.org/en-US/docs/Web/Web_Components